test/banners/desktop/components/BannerVar.spec.ts

```diff
@@ -87,7 +87,7 @@ describe( 'BannerVar.vue', () => {
 	describe( 'Donation Form Happy Paths', () => {
 		test.each( [
 			[ 'expectMainDonationFormGoesToUpgrade' ],
```

Comment:
I think you can add the `expectUpgradeToYearlyFormSubmitsUpgrade` and `expectUpgradeToYearlyFormSubmitsUpgrade` here?